targets/TARGET_NUVOTON/TARGET_M261/device/StdDriver/m261_timer_pwm.c

```diff
@@ -0,0 +1,451 @@
+/**************************************************************************//**
+ * @file     timer.c
+ * @version  V3.00
+ * @brief    Timer PWM Controller(Timer PWM) driver source file
+ *
+ * @copyright (C) 2019 Nuvoton Technology Corp. All rights reserved.
+*****************************************************************************/
```

Comment:
Looks like StdDriver is missing the acceptable license. Please add it (requested previously by @SeppoTakalo )

Reply:
@0xc0170 , @SeppoTakalo ,
StdDriver come from SDK, so we try to keep the same originally for easy maintenance & upgrade.
After refer to other targets,  could you accept the license as below:
```
  @copyright (C) 2019 Nuvoton Technology Corp. All rights reserved.
  
   Redistribution and use in source and binary forms, with or without modification,
   are permitted provided that the following conditions are met:
     1. Redistributions of source code must retain the above copyright notice,
        this list of conditions and the following disclaimer.
     2. Redistributions in binary form must reproduce the above copyright notice,
        this list of conditions and the following disclaimer in the documentation
        and/or other materials provided with the distribution.
     3. Neither the name of Nuvoton Technology Corp. nor the names of its contributors
        may be used to endorse or promote products derived from this software
        without specific prior written permission.
  
   THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
   AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
   IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
   DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
   FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
   DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
   SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
   CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
   OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
   OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```

Reply:
Looks fine to me (BSD-3 clause license as I understand it).

Reply:
@0xc0170 , @SeppoTakalo ,
Added acceptable license into std-drivers & register header files in commit a1e8122.
So, all the target relative files should fulfill your request.